docs/sidebar.yml

```diff
@@ -74,7 +74,7 @@ subsection:
           - page: reference/metaprogramming/staging.md
           - page: reference/metaprogramming/reflection.md
           - page: reference/metaprogramming/tasty-inspect.md
-          - page: reference/metaprogramming/simple-smp.md
+          # - page: reference/metaprogramming/simple-smp.md  description of a simplified metaprogramming language, this might not be the best place for it
```

Comment:
```suggestion
          - page: reference/metaprogramming/simple-smp.md  # description of a simplified metaprogramming language, this might not be the best place for it
            hidden: true
```